### PR TITLE
test: add new test 'Element Handle Select Text', Fixed webkit windows…

### PR DESCRIPTION
**Pull request by ephung01**
1. add new test - Test Element Handle Select Text scenarios
2. Fixed webkit windows platform check SSL Error message with regular expression in TestPageWaitForNavigation.shouldWorkWithClickingOnLinksWhichDoNotCommitNavigation()

### CODE REVIEW

File: playwright/src/test/java/com/microsoft/playwright/TestPageWaitForNavigation.java [hunk 1]
```diff
@@ -78,9 +77,8 @@ void shouldWorkWithClickingOnLinksWhichDoNotCommitNavigation() throws Interrupte
       fail("did not throw");
     } catch (PlaywrightException e) {
       // TODO: figure out why it is inconsistent on Linux WebKit.
-      assertTrue(e.getMessage().contains(expectedSSLError(browserType.name())) ||
-          (isWebKit() && getOS() == Utils.OS.LINUX && "Server required TLS certificate".equals(e.getMessage())),
-        "Unexpected exception: '" + e.getMessage() + "'");
+      System.out.println(e.getMessage());
```
Comment (yury-s) [inline playwright/src/test/java/com/microsoft/playwright/TestPageWaitForNavigation.java:80]: Revert this, you can pass it as second argument to assertTrue for debugging

File: playwright/src/test/java/com/microsoft/playwright/TestElementHandleSelectText.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  @Test
+  void shouldWaitForVisible() {
+    page.navigate(server.PREFIX + "/input/textarea.html");
+    ElementHandle textarea = page.querySelector("textarea");
+    textarea.evaluate("textarea => textarea.value = 'some value'");
+    textarea.evaluate("e => e.style.display = 'none'");
+
+    AtomicBoolean done = new AtomicBoolean(false);
+    // Async look for element timeout 3 seconds and return status if found or not
+    CompletableFuture<Boolean> completableFuture = CompletableFuture.supplyAsync(() -> {
```
Comment (yury-s) [inline playwright/src/test/java/com/microsoft/playwright/TestElementHandleSelectText.java:70]: Playwright is not thread safe, it's designed to work on a single thread. I'll probably add a check that the APIs are called only from one thread.

These kinds of tests (should wait for ..) are essentially testing backend behavior. In typescript they are very easy to implement with promises but it's much more difficult with sync API. Given that there is a good coverage for the functionality upstream I was skipping such tests in java. Alternatively, if you really want to port such tests they can be converted to trigger the condition asynchronously from inside the page but in this case I suggest you drop this test in java (just leave an empty method similar to `shouldWaitForDisplayNoneToBeGone`)

Comment (ephung01) [reply playwright/src/test/java/com/microsoft/playwright/TestElementHandleSelectText.java:70]: OK got it.  I will skip the last test case

File: playwright/src/test/java/com/microsoft/playwright/TestPageWaitForNavigation.java (hunk 1 above)
Comment (ephung01) [reply playwright/src/test/java/com/microsoft/playwright/TestPageWaitForNavigation.java:80]: This test will failed on Webkit (Windows & Linux) due to different text in exception message.   Also in Mac OS webkit, I think issue probably due to multi-lines exception return so when check with contain text not found - should use regex to resolve it.

Comment (yury-s) [reply playwright/src/test/java/com/microsoft/playwright/TestPageWaitForNavigation.java:80]: What I meant was to revert debug printing line `System.out.println(e.getMessage());`, the reset is fine.

**Pull request merged**